multiplayer/src/components/HeaderBar.tsx

```diff
@@ -0,0 +1,147 @@
+/// <reference path="../../../localtypings/react.d.ts" />
+
+// TODO multiplayer : Reduce duplication with skillmap HeaderBar.tsx (may require enabling tailwind css there or removing it here?)
```

Comment:
Interesting point. The reuse of any component authored here requires adopting tailwind everywhere... or for us to stop using it and go the route of custom class encrustation (noooo...). Note to self: Set up an engineering discussion to decide this.